docs/api/data-views/create.asciidoc

```diff
@@ -44,58 +41,135 @@ the data view is stored. The default is `false`.
 [[data-views-api-create-example]]
 ==== Examples
 
-Create a data view with a custom title:
+Create a data view to view data in the `logstash-*` indices:
+
+[source,sh]
+--------------------------------------------------
+$ curl -X POST api/data_views/data_view
+{
+  "data_view": {
+     "title": "logstash-*",
+     "name": "My Logstash Data View"
+  }
+}
+--------------------------------------------------
+// KIBANA
+
+Use the `data_view.fieldFormats` property to create custom field formats:
 
 [source,sh]
 --------------------------------------------------
 $ curl -X POST api/data_views/data_view
 {
   "data_view": {
-     "title": "hello"
+    "title": "logstash-*",
+    "name": "My Logstash data view 2",
+    "fieldFormats": {
+      "event_time": {
+        "id": "date_nanos"
+      },
+      "machine.ram": {
+        "id": "number",
+        "params": {
+          "pattern": "0,0.[000] b"
+        }
+      }
+    }
   }
 }
 --------------------------------------------------
 // KIBANA
 
-Customize the creation behavior:
+Use the `data_view.fieldAttrs` property to create custom labels:
 
 [source,sh]
 --------------------------------------------------
 $ curl -X POST api/data_views/data_view
 {
-  "override": false,
-  "refresh_fields": true,
   "data_view": {
-     "title": "hello"
+    "title": "logstash-*",
+    "name": "My Logstash data view 3",
+    "fieldAttrs": {
+      "utc_time": {
+        "customLabel": "Time (UTC)"
+      }
+    }
   }
 }
 --------------------------------------------------
 // KIBANA
 
-At creation, all data view fields are optional:
+Use the `data_view.runtimeFieldMap` property to create a data view with {ref}/runtime-search-request.html[runtime fields]:
 
 [source,sh]
 --------------------------------------------------
 $ curl -X POST api/data_views/data_view
 {
   "data_view": {
-      "id": "...",
-      "version": "...",
```

Comment:
Note: `version` has not been retained in the documentation. I am not sure if is harm in letting users update  `version`, but I can't think of how to document this field since I don't know if users will ever want or need to update it.